conversions/Backwards/src/main/java/com/agentdid127/resourcepack/backwards/BackwardsPackConverter.java

```diff
@@ -17,77 +17,90 @@
 public class BackwardsPackConverter extends PackConverter {
     Path INPUT_DIR;
 
-    public BackwardsPackConverter(String from, String to, String light, boolean minify, Path input, boolean debug, PrintStream out) {
+    public BackwardsPackConverter(String from, String to, String light, boolean minify, Path input, boolean debug,
```

Comment:
This breaks functionality with the GUI program

Reply:
I don't see visually where the issue is & I already fixed that by adding the Logger